### PR TITLE
Spruce up the Readme

### LINKED ISSUE #11: Improve the README
It's looking a bit bare-bones at the moment.  :)

### PR DESCRIPTION
**Pull request by CodaFi**
Fixes #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,126 @@
...
+/// This program acts like `cat` by blitting any input from stdin to stdout.
+runEffect <| stdinLn() >-> stdoutLn()
+```
+
+# System Requirements
+
+Aquifer supports OS X 10.9+ and iOS 7.0+.
+
+# Further Reading
+
```
Comment (CodaFi) [inline README.md]: Any suggestions?

File: README.md [hunk 2]
```diff
@@ -1,3 +1,126 @@
 # Aquifer
 
-This is a port of [pipes](http://hackage.haskell.org/package/pipes) (and its immediate associated ecosystem) to Swift, using [Swiftz](https://github.com/typelift/Swiftz) and [Focus](https://github.com/typelift/Focus).
+This is a port of [pipes](http://hackage.haskell.org/package/pipes) (and its 
+immediate associated ecosystem) to Swift, using 
+[Swiftz](https://github.com/typelift/Swiftz) and 
+[Focus](https://github.com/typelift/Focus).
+
+For those already familiar with the Haskell library, check out the source.  For
```
Comment (pthariensflame) [inline README.md]: Should "library" be "libraries" here?

Comment (CodaFi) [inline README.md]: Yes it should

File: README.md [hunk 3]
```diff
@@ -1,3 +1,126 @@
...
+    UI ==>       ==> DO ===/ \ UI==>       ==> NO    UI ==>       ==> NI
+        |    |    |           \   |    |    |            |    |    |
+        +----|----+            \  +----|----+            +----|----+
+             v                  \      v                      v
+             FR                  \====>DI                     FR
+```
+
+If that all seems complicated, don't worry!  The diagrams are always accompanied
+by proper explanations of the semantics of their respective methods in *plain
+english*.  If the documentation is ever unclear, feel free to ask a question,
```
Comment (pthariensflame) [inline README.md]: The word "English" needs to be capitalized, always (except in golf).

File: README.md (hunk 1 above)
Comment (pthariensflame) [inline README.md]: No idea.  We could point them towards to original pipes tutorial, but I don't know how helpful it would be.

Comment (CodaFi) [inline README.md]: Maybe it's best to leave it out, then.  Any idea what to put in its place if anything?

**Comment by CodaFi** [comment]
If there's nothing else, 

Ready for Review

**Comment by pthariensflame** [comment]
:+1:

**Comment by CodaFi** [comment]
:boat:

**Pull request merged**